homeassistant/components/huawei_lte/__init__.py

```diff
@@ -594,7 +603,10 @@ def should_poll(self) -> bool:
     @property
     def device_info(self) -> Dict[str, Any]:
         """Get info for matching with parent router."""
-        return {"connections": self.router.device_connections}
+        return {
+            "identifiers": self.router.device_identifiers,
```

Comment:
What is the benefit of having both connections and identifiers?

Reply:
Mac might not work reliably depending on network setup. Serial is not available in unauthenticated mode. Having both increases likelihood of success.